foundry.toml

```diff
@@ -110,3 +110,6 @@
   optimism = "https://optimism-mainnet.infura.io/v3/${API_KEY_INFURA}"
   polygon = "https://polygon-mainnet.infura.io/v3/${API_KEY_INFURA}"
   sepolia = "https://sepolia.infura.io/v3/${API_KEY_INFURA}"
+  arbitrum_sepolia = "https://arbitrum-sepolia.blockpi.network/v1/rpc/public"
```

Comment:
It would be a good idea to order them alphabetically so that the both of the versions of the same chain are put together.